homeworks/andrii.stoliarov_lightwarrior/8-components-templates/angular-gallery/src/app/app.component.html

```diff
@@ -0,0 +1,2 @@
+<h1>{{title}}</h1>
```

Comment:
```suggestion
<h1>Gallery</h1>
```
Якщо `title` не є динамічним то і робити для нього інтерполяцію не потрібно

Reply:
fixed